src/main/java/ca/corefacility/bioinformatics/irida/ria/web/linelist/LineListController.java

```diff
@@ -211,7 +218,8 @@ private AgGridColumn mapFieldToColumn(MetadataTemplateField field) {
 			int index = allFieldsLabels.indexOf(field.getFieldKey());
 			allFieldsAgGridColumns.remove(index);
 			allFieldsLabels.remove(index);
-			templateAgGridColumns.add(mapFieldToColumn(field));
+			// NEed to add parameter for if they have permissions to edit.
```

Comment:
`NEed` typo

Reply:
Fixed in 090463c35e2e98f0fdd78c6b1c7b7e3bda3bb59a